test/fixtures/loans.ts

```diff
@@ -79,6 +77,9 @@ export const createLoan = async (
   loanAmount: string,
   borrower: Signer
 ): Promise<string> => {
+  const amount = BigNumber.from(loanAmount)
+    .mul(BigNumber.from('10').pow('18'))
+    .toString()
```

Comment:
We should grab the lending token from the market and fetch the decimals instead of defaulting to `18`

Reply:
Also can use the new `BN` helper

Reply:
Good point, added in